integrations/nlu-rasa/src/RasaNlu.ts

```diff
@@ -49,4 +96,14 @@ export class RasaNlu extends NluPlugin<RasaNluConfig> {
   ): Promise<AxiosResponse<RasaResponse>> {
     return axios.post(`${this.config.serverUrl}${this.config.serverPath}`, { text }, config);
   }
+
+  private mapAlternativeIntents(altIntents: RasaIntent[]) {
+    // remove first element, because its the classified intent
+    altIntents.splice(0, 1);
+    const cutoff = this.config.alternativeIntents?.confidenceCutoff ?? 1.0;
```

Comment:
Instead of using a default value here, it is recommended to use `getDefaultConfig` and use the config-value directly.
In this example it would mean, that `RasaConfig.alternativeIntents` and `RasaConfig.alternativeIntents.confidenceCutoff` become required and `getDefaultConfig` returns `alternativeIntents.confidenceCutoff` as well.

Reply:
adapted in consequence of the above